forem/forem
# Include search term in title for improved accessibility

## What type of PR is this? (check all applicable)

Not sure. Feature? 🤷🏻‍♀️
Maybe an "accessibility" category would work here


## Description
This PR aims to add the search term to the page title (`h1`, not meta title), as per recommendations from Deque and accessibility best practices.

## Related Tickets & Documents
#14735 

## QA Instructions, Screenshots, Recordings
- go to your dashboard
- type something into the search input and hit Enter
- verify that the page title reads "Search results for [your search term]"
- delete your search term and hit Enter again
- verify that the page title reads "Search results"

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
